bioptim/dynamics/dynamics_functions.py

```diff
@@ -415,7 +447,7 @@ def muscles_driven(
         if fatigue is not None and "muscles" in fatigue:
             dxdt = fatigue["muscles"].dynamics(dxdt, nlp, states, controls)
 
-        return dxdt
+        return dxdt, MX(nlp.states.shape, ddq.shape[1])
```

Comment:
Does this actually work, or the MX is a place holder? If so, please make sure we raise an `NotImplementedError` somewhere so noone tries to use this dynamic function

Reply:
The flag is hidden in the collocation integrator, it allows to switch from the new implicit constraint to the older one when MX=0 is detected.